internal/e2e/rollup/package.json

```diff
@@ -1,5 +1,8 @@
 {
   "devDependencies": {
-    "jasmine": "~2.8.0"
+    "jasmine": "~2.8.0",
+    "rollup": "0.52.3",
```

Comment:
Are these needed? commonjs could go for sure as its not used. ```  npm_install(
      name = "build_bazel_rules_nodejs_rollup_deps",
      package_json = "@build_bazel_rules_nodejs//internal/rollup:package.json",
  )``` should handle rollup?

Reply:
Ah, but that one doesn't have jasmine.
I thought "oh, no problem, I could union them, I'll make a filegroup with a 
`srcs=['@bbrj_rollup_deps//:node_modules', '@bbrj//internal/e2e/rollup:node_modules']`
but this doesn't work because the resulting filegroup isn't flattened

Reply:
Ohh. Ok. That makes sense. commonjs could go still.

Reply:
oh, right commonjs was still here, fixed